Dashboard/app/js/lib/components/devices/AssignmentsList.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+/* eslint-disable no-nested-ternary */
+const AssignmentsList = ({
+  assignments,
+  onEdit,
+  onDelete,
+  onSort,
+}) => (
+  <div>
+    <div className="deviceControls">
+      <a
+        className="btnOutline"
+        onClick={() => onEdit('new')}
+        onKeyDown={() => onEdit('new')}
+        style={{ float: 'right' }}
+      >
+        {Ember.String.loc('_create_new_assignment')}
+      </a>
+    </div>
+    <div
+      id="devicesListTable_length"
```

Comment:
generally id's are not needed in react. You can remove this if youre not using the id elsewhere in the codebase